hoomd/hpmc/integrate.py

```diff
@@ -133,16 +134,23 @@ def __init__(self, seed, d, a, translation_move_probability, nselect):
         typeparam_fugacity = TypeParameter('depletant_fugacity',
                                            type_kind='particle_types',
                                            param_dict=TypeParameterDict(
-                                               0., len_keys=1))
+                                               0., len_keys=2)
```

Comment:
Update the documentation for `depletant_fugacity` to state that it takes two types and explain meaning of fugacity as a type matrix.

Reply:
I clarified as suggested that the feature is temporary and will be removed in the subsequent PR, where the fugacity will be defined per type as before. As explained above, the type pair option was meant to support depletants that behave as particle type `A` when overlapping with another particle, and as particle type `B` when overlapping with the particle that particle is overlapping with. This treatment is not consistent with the many body character of the depletion interaction and is only correct when there are no higher order than pair interactions in the system.

Reply:
Understood. The betas are unstable and its OK if we make and unmake breaking changes. I may delay merging this until after the beta 4 release hoping that the next PR could be merged before beta 5.